auth_otp/services/auth_service.py

```
@@ -0,0 +1,437 @@
+
+logger = logging.getLogger(__name__)
+User = get_user_model()
+
+
+class AuthService:
+    """
+    سرویس مدیریت احراز هویت و توکن‌ها
+    """
+    
+    @staticmethod
+    def create_user_if_not_exists(phone_number: str, user_type: str = 'patient') -> Tuple[User, bool]:
```

Ruff (0.12.2)

Use `tuple` instead of `Tuple` for type annotation

Replace with `tuple`

(UP006)